pkg/manager/member/ticdc_upgrader.go

```diff
@@ -108,7 +108,14 @@ func (u *ticdcUpgrader) Upgrade(tc *v1alpha1.TidbCluster, oldSet *apps.StatefulS
 			}
 			continue
 		}
-		mngerutils.SetUpgradePartition(newSet, i)
+
+		err = gracefulShutdownTiCDC(tc, u.deps.CDCControl, u.deps.PodControl, pod, ordinal, "Upgrade")
+		if err != nil {
+			return err
```

Comment:
add some description about the error

Reply:
We log details in `gracefulShutdownTiCDC`, see sample logs below.

```log
I0801 08:49:25.977589       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"default", Name:"basic", UID:"16034049-84c9-4749-bde4-3008e10b59f5", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"2458961", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/basic-ticdc-3436353 for controller TidbCluster/basic successfully
I0801 08:49:25.987193       1 stateful_set_control.go:94] : [default/basic]'s StatefulSet: [default/basic-ticdc] updated successfully
I0801 08:49:26.011110       1 tidbcluster_control.go:69] TidbCluster: [default/basic] updated successfully
I0801 08:49:26.949297       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:49:26.949343       1 ticdc_scaler.go:195] ticdc.Upgrade: begin graceful shutdown basic-ticdc-2 in cluster default/basic
I0801 08:49:26.966106       1 pod_control.go:87] Pod: [default/basic-ticdc-2] updated successfully, : [default/basic]
I0801 08:49:26.966137       1 ticdc_scaler.go:207] ticdc.Upgrade: set pod basic-ticdc-2 in cluster default/basic annotation tidb.pingcap.com/ticdc-graceful-shutdown-begin-time to 2022-08-01T08:49:26Z successfully
I0801 08:49:27.124554       1 tidbcluster_control.go:69] TidbCluster: [default/basic] updated successfully
I0801 08:49:27.125517       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 is still the owner, try resign owner again, requeuing
I0801 08:49:28.548035       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:49:29.892315       1 tidbcluster_control.go:69] TidbCluster: [default/basic] updated successfully
I0801 08:49:29.892351       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 131 tables, wait draining, requeuing
I0801 08:49:30.147198       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:49:31.534393       1 tidbcluster_control.go:69] TidbCluster: [default/basic] updated successfully
I0801 08:49:31.534444       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 131 tables, wait draining, requeuing
I0801 08:49:31.747176       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:49:32.873400       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 131 tables, wait draining, requeuing
I0801 08:49:33.547769       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:49:34.871809       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 131 tables, wait draining, requeuing
I0801 08:49:36.547747       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:49:37.872539       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 131 tables, wait draining, requeuing
I0801 08:49:40.951512       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:49:42.072695       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 121 tables, wait draining, requeuing
I0801 08:50:06.049524       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:50:07.272154       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 101 tables, wait draining, requeuing
I0801 08:50:36.055293       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:50:37.072982       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 81 tables, wait draining, requeuing
I0801 08:50:46.159003       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:50:47.272882       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 71 tables, wait draining, requeuing
I0801 08:51:06.070187       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:51:07.273668       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 61 tables, wait draining, requeuing
I0801 08:51:36.052212       1 statefulset.go:169] set default/basic-ticdc partition to 3
I0801 08:51:37.076171       1 tidb_cluster_controller.go:124] TidbCluster: default/basic, still need sync: ticdc.Upgrade: cluster default/basic basic-ticdc-2 still has 31 tables, wait draining, requeuing
```